tools/mo/openvino/tools/mo/front/caffe/scale_ext.py

```diff
@@ -25,16 +23,16 @@ def extract(cls, node):
         
         if model is None and len(pb.bottom) == 1:
             # default weights and biases for scale layer if the caffemodel file doesn't contain them
-            model = NamedAttrsClass({'blobs': np.array([NamedAttrsClass({'data': np.array([1])}),
-                                                 NamedAttrsClass({'data': np.array([0])})])})
+            model = NamedAttrsClass({'blobs': mo_array([NamedAttrsClass({'data': mo_array([1])}),
```

Comment:
`mo_array` is not lucky name. We have `int64_array` and why is it called `mo_array` for int32. Please revert this change

Reply:
mo_array has a special purpose. It acts as np.array, but if no dtype provided and resulting dtype is fp64 it forces fp32 dtype. The name was agreed with @jane-intel

Reply:
I am fine to have this function but I don't support to change the code overall. Please revert this change and use `mo_array` for your particular task only

Reply:
what if in some code we have a list x = [1,2,3] and I call np.array(x). So the result will be np.array of np.int32
In your case, if we substitute this with mo_array(x), the result will be np.array of np.float32.

This function looks dangerous to me

Reply:
> what if in some code we have a list x = [1,2,3] and I call np.array(x). So the result will be np.array of np.int32 In your case, if we substitute this with mo_array(x), the result will be np.array of np.float32.
> 
> This function looks dangerous to me

If result of np.array is int32 the result of mo_array will be int32 too. Only in one case it will be different. If result of np.array is fp64 and type wasn't provided, which means fp64 wasn't expected.